src/main/java/sx/blah/discord/api/internal/DispatchHandler.java

```diff
@@ -417,7 +417,7 @@ private void messageUpdate(MessageObject json) {
 			client.dispatcher.dispatch(new MessagePinEvent(toUpdate));
 		} else if (oldMessage.getEmbedded().size() < toUpdate.getEmbedded().size()) {
 			client.dispatcher.dispatch(new MessageEmbedEvent(toUpdate, oldMessage.getEmbedded()));
-		} else if (json.content != null && oldMessage.getContent().equals(json.content)) {
+		} else if (json.content != null) {
```

Comment:
Don't remove `oldMessage.getContent().equals(json.content)`, it should have a ! in front of that statement which is causing the issues